src/views/AuthoritiesSearch/AuthoritiesSearch.test.js

```diff
@@ -225,6 +225,16 @@ describe('Given AuthoritiesSearch', () => {
 
         expect(exportRecordsButton).not.toBeInTheDocument();
       });
+
+      it('should not show Save Authorities UUIDs button', () => {
+        renderAuthoritiesSearch();
+
+        fireEvent.click(screen.getByRole('button', { name: 'stripes-components.paneMenuActionsToggleLabel' }));
+
+        const saveAuthoritiesUIIDSButton = screen.queryByRole('button', { name: 'ui-marc-authorities.actions.saveAuthoritiesUIIDS' });
+
+        expect(saveAuthoritiesUIIDSButton).toBeNull();
```

Comment:
`not.toBeInTheDocumemt()` can be more meaningful than asserting a query result is null, I would recommend it for the most explicit and user-centric approach